model/metadata.cpp

```diff
@@ -493,6 +497,45 @@ model::metadata::is_exclusive(void) const
 }
 
 
+/// Returns execution environment name.
+///
+/// \return Name of configured execution environment.
+const std::string&
+model::metadata::execenv(void) const
+{
+    if (_pimpl->props.is_set("execenv")) {
+        return _pimpl->props.lookup< config::string_node >("execenv");
+    } else {
+        return get_defaults().lookup< config::string_node >("execenv");
+    }
```

Comment:
A minor control flow simplification:
```suggestion
    }
    return get_defaults().lookup< config::string_node >("execenv");
```

Reply:
It simply repeats the same style of the existing functions around, so it's easier to read all of them -- when the eyes quickly catch the same structure. But why not, let's do it.

Reply:
Actually, the current style guide (https://github.com/freebsd/kyua/blob/master/CONTRIBUTING.md#style-guide) asks to watch the existing style of braces etc. Probably, it's better to keep it as is, and some future refactoring could simplify the flow for all the places at once. What do you think?